master_db.py

```diff
@@ -54,16 +57,28 @@
                                         "name": "nsample", "type":["int", "List[int]"]}}]  
                 }
 
-    for name in distributions:
+    print("Insertando distribuciones...")
+    for name, parameters in distributions.items():
         try:
-            distribution = Distribution()
-            distribution.name = name
-            distribution.type = parameters[name]
+            distribution = MasterDistribution(
+                name=name,
+                type=parameters).save()
 
-            distribution.save()
+            print(f"distribución {name} insertada correctamente")
+        except Exception as error:
+            print(f"No fue posible insertar la distribución {name}")
+            print(error)
```

Comment:
Same

Delete all prints, you could set this
```suggestion
        except Exception as error:
            raise Exception(error)
```